src/Fixer/ClassNotation/NullPropertyDeclarationFixer.php

```diff
@@ -66,10 +66,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $index = $tokens->getNextMeaningfulToken($index);
 
                 if ($tokens[$index]->equals('=')) {
-                    $valueTokenIndex = $index = $tokens->getNextMeaningfulToken($index);
+                    $index = $tokens->getNextMeaningfulToken($index);
 
                     if ($tokens[$index]->equals([T_STRING, 'null'], false)) {
-                        $tokens->clearRange($varTokenIndex + 1, $valueTokenIndex);
+                        for ($i = $varTokenIndex + 1; $i <= $index; ++$i) {
+                            if (!$tokens[$i]->isComment() && !($tokens[$i]->isWhitespace() && false !== strpos($tokens[$i]->getContent(), "\n"))) {
```

Comment:
I would swap the whitespace and the comment checks: whitespaces are more likely to be found here than comments, this should make this `if` slightly faster in such case.

Also, I would remove whitespaces even when they contain a newline (except when preceded by a single-line comment), WDYT?